lib/config.js

```diff
@@ -47,25 +55,18 @@ function _resetConfig () {
   _config = undefined
 }
 
-function getConfig (dir) {
+function getConfig (dir, argv, rootPkg) {
   if (_config) return BB.resolve(_config)
-  return readConfig().then(config => {
-    _config = {
-      config,
+  return readConfig(argv).then(config => {
+    log.level = config['loglevel']
+    _config = Object.assign(config, {
+      userAgent: config['user-agent'] || `${cipmPkg.name}@${cipmPkg.version} ${process.release.name}@${process.version.replace(/^v/, '')} ${process.platform} ${process.arch}`,
       dir,
+      prefix: dir,
       failOk: false,
-      force: config.force,
-      group: config.group,
-      ignorePrepublish: config['ignore-prepublish'],
```

Comment:
The options in lifecycle are all camelcased. This will cause ignorePrepublish to not be respected.